packages/components/src/utils/getReferenceLinkHref.ts

```diff
@@ -31,3 +51,16 @@ export const getReferenceLinkHref = (
 
   return refPage.permalink
 }
+
+export const getReferenceLinkHref = (
+  referenceLink: string | undefined,
+  sitemap: IsomerSitemap,
+) => {
+  if (!referenceLink) {
+    return undefined
+  }
+
+  const actualLink = getActualLinkFromReference(referenceLink, sitemap)
```

Comment:
should we inline this function? feels unnecessary to extract it out when its only used in a single place